spinalcordtoolbox/scripts/sct_dmri_separate_b0_and_dwi.py

```diff
@@ -194,7 +194,13 @@ def main(argv=None):
     # Average b=0 images
     if average:
         printv('\nAverage b=0...', verbose)
-        run_proc(['sct_maths', '-i', b0_name + ext, '-o', b0_mean_name + ext, '-mean', 't'], verbose)
+        img = Image(b0_name + ext)
+        out = img.copy()
+        dim_idx = 3
+        if len(np.shape(img.data)) < dim_idx + 1:
+            raise ValueError("Expecting image with 4 dimensions!")
```

Comment:
Is this the right check for `-mean t`? See the previous review comment, which has a different check for `-mean t`.

Reply:
Comment: https://github.com/neuropoly/spinalcordtoolbox/pull/2975#discussion_r535385495

> If the condition if dim_idx + 1 > len(np.shape(img.data)): ever isn't true, then there's a bigger problem... (so just appending a new axis would be hiding that).

Is that not what you were proposing?

Reply:
I apparently have the memory of a goldfish. Thank you for reminding me. :sweat_smile: